presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -140,11 +140,11 @@ will function as both a coordinator and worker, use this configuration:
     discovery-server.enabled=true
     discovery.uri=http://example.net:8080
 
-If single coordinator is not sufficient, disaggregated coordinator setup can be used which supports multiple coordinator using below minimal configuration:
+If a single coordinator is not sufficient, disaggregated coordinator setup can be used which supports multiple coordinators using the following minimal configuration:
```

Comment:
```suggestion
If a single coordinator is not sufficient, use a disaggregated coordinator setup which supports multiple coordinators using the following minimal configuration:
```
Nit to add "a" here as well, suggest change to active voice.

Reply:
done